### PR TITLE
Updated conf dir

### PR DESCRIPTION
**Pull request by mizzioisf**
Updated documentation according to new directory layout
/etc -> /data/conf
Other updates and fixes

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -1073,7 +1073,7 @@ The default settings for a GSM modem should work with the majority of GSM device
 The exception is the PORT parameter which must be changed to match the SERIAL port address used by the device plugged into the system.
 
 If the PORT param is correct but Open Hospital is not able to communicate with the device (try switching to DEBUG log level – or lower - during this setup –
-see <<log4j-properties,log4j.properties>>) and try to use the SetupGSM utility (OH startup script with the "G" option: oh.sh -G / oh.bat -> select "G").
+see <<log4j-properties,log4j.properties>>) and try to use the SetupGSM utility (launch OH startup script with the "G" option: oh.sh -G / oh.bat -> select "G").
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: missing determiner:  launch OH startup script -> launch the OH startup script

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -389,11 +389,11 @@ include::https://raw.githubusercontent.com/informatici/openhospital/master/oh-bu
 
 Backup of the Open Hospital database can be performed in several ways.
 
-NOTE: Starting with version 1.11, the easiest way is to use the provided scripts (oh.sh/oh.bat) by choosing the option "-s".
+NOTE: Starting with version 1.11, the easiest way is to use the provided scripts: on Linux execute *oh.sh -s*, on Windows launch *oh.bat* and choose the option *s*.
 
 Restore of the Open Hospital database can be performed also in several ways.
 
-NOTE: Starting with version 1.11, the easiest way is to use the provided scripts (oh.sh/oh.bat) by choosing the option "-r".
+NOTE: Starting with version 1.11, the easiest way is to use the provided scripts: on Linux execute *oh.sh -r*, on Windows launch *oh.bat* and choose the option *r*.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: Didn't we agree not to use "Starting with version x.x"?   If we didn't then there are other places this needs to be added; for example, the DBMS configuration files were previously in *etc* and now they are in *data/db*.

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Fixed, thanks.

File: doc_admin/AdminManual.adoc (hunk 2 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: I already corrected the reference to the data/conf directory in place of the etc folder.
Agree on the 1.11.1 ref version, removed. We will still have to do another update before the 1.11.1 release to replace the 1.11.0 references still present.

Comment (dbmalkovsky) [reply doc_admin/AdminManual.adoc]: My point was more if we are highlighting changes in options to scripts happening why aren't we highlighting when changes take place to the moving of locations of files?   I see both types of changes happening at a particular release boundary but only one of them is specifically noted.     Just my 2 cents.

Comment (mwithi) [reply doc_admin/AdminManual.adoc]: I think to remove all refs in any case is the best thing and also more easy to manage.

**Pull request merged**